### PR TITLE
Separate top and bottom content for Next

### PR DESCRIPTION
**Pull request by sumeetadur**

### CODE REVIEW

**Comment by sumeetadur** [comment]
@callumlocke @kavanagh can you look through this & let me know if I need to change or add anything? Thanks

**Comment by kavanagh** [comment]
I'm not a fan of the naming: 'top' and 'bottom'. Is there something more descriptive?

~~Also do we want to separate the card styles (eg white bg with shadow or grey bg without) from the styles that layout of the content of the card?~~ EDIT: I see, looks like you already did this, that absense of the BEM modifiers threw me.

File: sass/card-bottom.scss [hunk 1]
```diff
@@ -19,6 +20,28 @@
 		}
 	}
 
+	.content {
```
Comment (kavanagh) [inline sass/card-bottom.scss]: why not BEM?

File: sass/card-bottom.scss [hunk 2]
```diff
@@ -19,6 +20,28 @@
...
+		overflow: hidden;
+		padding-top: 20px;
+	}
+
+	&.bottom {
```
Comment (kavanagh) [inline sass/card-bottom.scss]: BEM.

**Pull request merged**